app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def show
+    @post = Post.find(params[:id])
+  end
+end
```

Comment:
Hi @TedLivist 

I hope you are doing well and thanks for the review, can you tell me why I need an index action or page for the post because I think my app working as expected as given in the required wire frame or did I miss something ??